morpheus/messages/multi_response_message.py

```diff
@@ -193,8 +165,7 @@ def get_output(self, name: str):
             Inference output.
 
         """
-
-        return self.memory.tensors[name][self.offset:self.offset + self.count, :]
+        return self.memory.get_output(name)[self.offset:self.offset + self.count, :]
```

Comment:
Its not in this diff, but `copy_ranges` has 2 arguments in the base class but 1 in the derived class